src/components/AnchorForAttachmentsOnly/AnchorForAttachmentsOnlyTypes.ts

```diff
@@ -0,0 +1,9 @@
+import {StyleProp, ViewStyle} from 'react-native';
+
+type AnchorForAttachmentsOnlyProps = {
+    source: string;
```

Comment:
Please add comments, you can reuse the one from removed [anchorForAttachmentsOnlyPropTypes.js](https://github.com/Expensify/App/pull/33429/files#diff-c1f581eca6e59cae5ca37b65ab3cc176b90ee3adac31047df5b341679ac21dc3) file